keyvi/include/keyvi/dictionary/completion/prefix_completion.h

```diff
@@ -173,6 +173,7 @@ class PrefixCompletion final {
           if (data->traverser) {
             TRACE("Current depth %d", exact_prefix + data->traverser.GetDepth() - 1);
 
+            // TODO(Amit) use int32_t instead of int.
```

Comment:
Can't we do the similar change to `GetScore()`, by just moving your `TODO` and type cast inside `Put()` method, and use `size_t` already here ?